Gemfile

```diff
@@ -30,7 +30,7 @@ group :development do
 end
 
 group :development, :test do
-  gem 'govuk-lint'
+  gem 'govuk-lint', '3.3.0'
```

Comment:
Should we do `~> 3.3.0` so we get subsequent compatible updates?

Reply:
Assuming we semvar it `'~> 3.3'` would be better

Reply:
Thanks @thomasleese @kevindew. Both of these will lock the Gemfile to 3.3.1 which contains a load of incompatible lint changes. So suggest this bump gets handled separately